assets/l10n/app_en.arb

```diff
@@ -19,6 +19,10 @@
   "@chooseAccountPageTitle": {
     "description": "Title for ChooseAccountPage"
   },
+  "switchAccountButton": "Switch account",
+  "@switchAccountButton": {
+    "description": "Main menu button for the choose-account page"
```

Comment:
```suggestion
    "description": "Label for main-menu button leading to the choose-account page."
```

The string isn't a button, but rather the label for a button.

Separately, without the hyphen it reads to me a bit more like: it's (a label for) a menu button for the choose-account page, namely the main such button.